downstream/modules/dev-guide/con-architecture-overview.adoc

```diff
@@ -0,0 +1,14 @@
+
+[id="con-architecture-overview_introduction"]
+
+
+= About content workflows
+
+
+[role="_abstract"]
+The following list shows the possible arrangements and uses of tools that will be available on the {PlatformNameShort} 2.0, along with how they can be utilized:
```

Comment:
> possible arrangements and uses of tools **that will be** available on the {PlatformNameShort} 2.0
> {{Navigator}} only -- can be used today in {PlatformNameShort} 1.2

Would it be correct to say "supported" here, rather than "possible"? 

Is this guide just for early access users? If not, maybe don't say "will be available" and "can be used today".

Reply:
updated